articles/task-scheduling.md

```diff
@@ -609,7 +609,7 @@ class Solution {
 ### Time & Space Complexity
 
 * Time complexity: $O(m)$
-* Space complexity: $O(m)$
+* Space complexity: $O(1)$
```

Comment:
Maybe i'm mistaken, but shouldn't the space complexity remain O(m)?

Oh, because the character set is 26. Can we add a note that this is because we only store a limited amount of values in the heap, like in: https://github.com/neetcode-gh/leetcode/commit/7a6051c623e905a01768115b1b8a5ba3963d1c32

Someone asked me a question about this the other day so i thought it was worth clarifying.

Reply:
The space complexity is O(1) because we store almost 26 elements only in any data structure.

It is a good idea to add that note. I will do that for all articles when I am adding the "Swift" code.